cmake/BuildLibCURL.cmake

```diff
@@ -70,6 +70,14 @@ macro(curl_build)
     endif()
     list(APPEND LIBCURL_CMAKE_FLAGS "-DENABLE_ARES=${ENABLE_ARES}")
 
+    # Setup http2 and nghttp2 library path
+    if(BUNDLED_LIBCURL_USE_NGHTTP2)
+        set(USE_NGHTTP2 "ON")
+    else()
+        set(USE_NGHTTP2 "OFF")
+    endif()
+    list(APPEND LIBCURL_CMAKE_FLAGS "-DUSE_NGHTTP2=${USE_NGHTTP2}")
```

Comment:
But here is nothing about paths.

I curious: cmake dependencies works here to setup correct linker path?

Reply:
I added `NGHTTP2_INSTALL_DIR` path to `CMAKE_FIND_ROOT_PATH` but I have doubts that it affects something, because without these changes, the result was the same.

Reply:
To sum up: when we don't set `CMAKE_FIND_ROOT_PATH` libcurl finds the system (dynamic) libnghttp2 library. At linking of the tarantool executable we pass our libnghttp2.a and everything works good. However, I guess, we'll meet a problem if there is no nghttp2 library in the system.

Anyway, it was fixed: now we pass the libnghttp2.a location within `CMAKE_FIND_ROOT_PATH`.